.storybook/3.COLORS.stories.mdx

```diff
@@ -0,0 +1,182 @@
+import { Meta } from '@storybook/addon-docs';
+import ActionaleMessage from '../ui/components/ui/actionable-message';
+import designTokenDiagramImage from './images/design.token.graphic.svg';
+
+<Meta title="Design Tokens / Color" />
+
+# Color
+
+Color is used to express style and communicate meaning.
+
+<ActionaleMessage
+  type="warning"
+  message="We are in the process of consolidating all of our colors, making them accessible and enabling theming. Many of the colors used throughout the codebase are deprecated please follow the guide below to ensure you are using the correct colors when building MetaMask UI"
+/>
+
+<br />
+
+## Design Tokens
+
+We are importing design tokens as CSS variables from [@metamask/design-tokens](https://github.com/MetaMask/design-tokens) repo to help consolidate colors and enable theming across all MetaMask products.
+
+### Token Tiers
+
+We follow a 3 tiered system for color design tokens and css variables.
+
+<div
+  style={{
+    textAlign: 'center',
+    backgroundColor: 'var(--color-background-alternative)',
+    padding: 32,
+  }}
+>
+  <img width="80%" src={designTokenDiagramImage} />
+</div>
+
+<br />
+<br />
+
+### **Brand colors** (tier 1)
+
+These colors **SHOULD NOT** be used in your styles directly they are used as a reference for the [Theme Colors](#theme-colors-tier-2). Brand colors should just keep track of every color used in our app.
+
+#### Example of brand color css variables
+
+```css
+/** !!!DO NOT USE BRAND COLORS DIRECTLY IN YOUR CODE!!! */
+  var(--brand-colors-white-white000)
+  var(--brand-colors-white-white010)
+  var(--brand-colors-grey-grey030);
+```
+
+### **Theme Colors** (tier 2)
+
+Theme colors are color agnostic, semantically neutral and theme compatible design tokens that you can use in your code and styles. Please refer to the description of each token for it's intended purpose in [`@metamask/design-tokens`](https://github.com/MetaMask/design-tokens/blob/main/src/figma/tokens.json#L329-L554).
```

Comment:
```suggestion
Theme colors are color agnostic, semantically neutral and theme compatible design tokens that you can use in your code and styles. Please refer to the description of each token for it's intended purpose in [@metamask/design-tokens](https://github.com/MetaMask/design-tokens/blob/main/src/figma/tokens.json#L329-L554).
```

What if we removed code quotes to enable the link style, or use code quotes for the `@metamask/design-tokens` instance on line 20? Same comment applies to 2 other `@metamask/design-tokens` instances below.

Reply:
Great suggestion. I will update. Thanks!

Reply:
Updated in https://github.com/MetaMask/metamask-extension/pull/13657/commits/bfbb6d6cdf98c91129ae2c256ce44383365357b8